### PR TITLE
Fix dependency:analyze violations

### PR DESCRIPTION
**Pull request by busjaeger**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,6 +75,11 @@
             <artifactId>micrometer-core</artifactId>
             <version>1.2.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>13.0</version>
```
Comment (horwitzj) [inline pom.xml]: 17.0.0?

Comment (horwitzj) [reply pom.xml]: (https://mvnrepository.com/artifact/org.jetbrains/annotations/17.0.0)

Comment (busjaeger) [reply pom.xml]: That's what `dependency:analyze` asked for. Let me try with 17.0.0

Comment (busjaeger) [reply pom.xml]: OK, seems to be happy with that as well.

**Pull request merged**